test/public/runs/overview.test.js

```diff
@@ -672,7 +672,7 @@ module.exports = () => {
     });
 
     it('should successfully filter on a list of run numbers and inform the user about it', async () => {
```

Comment:
Please add tests here on the substring filter, and same for the API test (runs api)